tests/geogr/test_distances.py

```diff
@@ -99,8 +137,17 @@ def test_meters_to_decimal_degrees(self):
 
         for degree, lat_output in input_result_dict.items():
             for lat, meters in lat_output.items():
-
                 decimal_degree_output = meters_to_decimal_degrees(meters, lat)
                 assert np.isclose(decimal_degree_output, degree, atol=0.1)
 
 
+class Testcalc_haversine_length_of_linestring:
+    def test_distance(self, gdf_ls):
+        dist = calc_haversine_length_of_linestrings(gdf_ls)
```

Comment:
Are there any asserts or checks here?

Reply:
I added a weak one but it is more to see if it runs through. (length calculation is tested in the function below).